config.go

```diff
@@ -67,4 +67,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Lables is the runtime itself specified information.
```

Comment:
Maybe “Labels holds runtime-specified information that is not structured by this specification.”?